src/pages/spawn/spawnHost/spawnHostModal/PublicKeyForm.tsx

```diff
@@ -1,22 +1,24 @@
-import React, { useReducer } from "react";
```

Comment:
I'm still encountering the issue i ran into when i tried to do this. 
<img width="670" alt="image" src="https://user-images.githubusercontent.com/4605522/156076299-1f9fd82b-d8ba-4c3c-8a1d-8103a44ddd3b.png">


The dropdown collides with the lg check box.